### PR TITLE
fix last parenthesis

### PR DESCRIPTION
**Pull request by borisjaskerovich**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

- test parameters for validating bids
```
{
  bidder: '<bidder name>',
  params: {
    // ...
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [ ] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

File: modules/memegenBidAdapter.js [hunk 1]
```diff
@@ -9,6 +9,12 @@ const openRtbAdapterHost = 'whichtalk.com';
 var timeout;
 var bidsRequested;
 
+String.prototype.removeCharAt = function (i) {
```
Comment (shlomisut) [inline modules/memegenBidAdapter.js:12]: seems like an overkill to remove the last char

Comment (borisjaskerovich) [reply modules/memegenBidAdapter.js:12]: it's not the last. i search for the last appearance of ")"

**Review by shlomisut — APPROVED** [review]

File: modules/memegenBidAdapter.js (hunk 1 above)
Comment (shlomisut) [reply modules/memegenBidAdapter.js:12]: which in this case, is the last..

**Comment by ghost** [comment]
Build Failed - please check http://jenkins.thetimmedia.site:8080/job/prebid_pull_request/117/

**Pull request merged**